plugins/prometheus/mana.go

```diff
@@ -43,10 +49,62 @@ func registerManaMetrics() {
 			Help: "Top percentile node belongs to in terms of consensus mana.",
 		})
 
+	averageAccessPledgeBM1 = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "mana_average_access_pledge_bm1",
+			Help: "Average base mana 1 of pledged access mana.",
+		},
+		[]string{
+			"nodeID",
+		})
+	averageAccessPledgeBM2 = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "mana_average_access_pledge_bm2",
+			Help: "Average base mana 2 of pledged access mana.",
+		},
+		[]string{
+			"nodeID",
+		})
```

Comment:
these two could be merged into one by introducing a new label "baseMana" which could either be "1" or "2".